ingester/ingester_lifecycle_test.go

```diff
@@ -61,6 +76,82 @@ func TestIngesterRestart(t *testing.T) {
 	})
 }
 
+func TestIngesterTransfer(t *testing.T) {
+	cfg := defaultIngesterTestConfig()
+
+	// Start the first ingester, and get it into ACTIVE state.
+	cfg1 := cfg
+	cfg1.id = "ingester1"
+	cfg1.addr = "ingester1"
+	cfg1.ClaimOnRollout = true
+	cfg1.SearchPendingFor = 10 * time.Second
```

Comment:
Why is this 10s and `JoinAfter` 60s? My hunch is that they're both code for "a really long time", but if that's the case why not pick a year?